metaflow/plugins/kfp/kfp_step_function.py

```diff
@@ -43,17 +41,18 @@ def kfp_step_function(
 
     cmd = cmd_template.format(
         run_id=kfp_run_id,
-        datastore_root=datastore_root,
         passed_in_split_indexes=passed_in_split_indexes,
     )
 
+    metaflow_configs_new = {
+        name: value for name, value in metaflow_configs.items() if value
```

Comment:
Since in function signature `metaflow_configs: Dict[str, str] = None` would you want to handle the case where metaflow_configs == None somewhere?

Reply:
it'll never be None, it's always fed in through kfp.py

Reply:
Shall we remove this default value from function signature then?

Reply:
done!